### ISSUE TITLE
Call to run migrations does not take into account sibling deployments in a pipeline

### ISSUE THREAD

**Issue by stevejalim** [yunojuno-archive/heroku-tools#10]
If you have more than one app in a pipeline's target node, only the one nominated in the `.herokutoolsconf` file will have its migrations run.

ie, in  

```
                /------- [FOO]
[UAT] ----------
                \------- [BAR]
```

if FOO is referenced in the conf, it has its migrations run, but BAR is ignored.

Suggestions

1) always run migrations - ie, use the buildpack defaults

2) use the Heroku API to get the pipleine target app names and pass that to the migration-running code

**Comment by hugorodgerbrown** [comment]
Yup - I had noticed this (and parked in the back of my brain). I think that running migrations all the time is probably the way to go. Now that we have got in the habit of running migrations without having to put up the maintenance page they seem less of an issue.

**Comment by stevejalim** [comment]
Looking into this more, I was mistaken re the buildpack - https://github.com/heroku/heroku-buildpack-python does not appear to include support for running migrations by default, only `collectstatic`.

I can make `heroku-tools` always run migrations, but that doesn't change the fact that it will only run them on the main app nominated in the configuration file -- it won't do any siblings in the pipeline.

So, two more suggestions:

1) Create a buildpack which does run migrations (and drop migration support from heroku-tools to save redundant work -- but this may not well serve all users of the proj)

OR

2) Use the Heroku API to [get the relevant platform data](https://devcenter.heroku.com/articles/pipelines-using-the-platform-api) and run the migration command on all the relevant apps in that stage


I prefer 2

**Comment by stevejalim** [comment]
Option 3 - use the `bin/post_compile` trick mentioned in this ticket https://github.com/heroku/heroku-buildpack-python/issues/233  

Option 4 - try the Release Phase beta: https://devcenter.heroku.com/articles/release-phase

However, neither option helps make `heroku-tools` better at handling this situation

**Comment by stevejalim** [comment]
Option 5: via a convo with @hugorodgerbrown directly - stick it in the configuration file

MUCH EASIER.

**Comment by hugorodgerbrown** [comment]
Closed by #13
